test-app/src/components/app.tsx

```diff
@@ -9,7 +9,19 @@ import { useAuth } from '@common/use-auth';
 import { GlobalStyles } from '@components/global-styles';
 
 export const App: React.FC = () => {
-  const { authOptions, state, authResponse, appPrivateKey, handleSignOut } = useAuth();
+  const { authOptions, state, authResponse, appPrivateKey, handleSignOut, handleIsOnboarding } =
+    useAuth();
+
+  const handleMessage = (event: MessageEvent) => {
+    if (event.data.method === 'completeOnboarding') {
+      handleIsOnboarding(true);
+    }
```

Comment:
I can't tell, does `isOnboarding` if true mean they've completed it, or that they need to `completeOnboarding`